operator keeps CrashLooping

**Describe the bug**
The operator is stuck in `CrashLoopBackOff` after fresh install on minikube.

The logs of the operator show this:
```
k -n mondoo-operator logs pod/mondoo-operator-controller-manager-c6c746d6c-885w8                                                                                                                   2022-09-22T09:55:45Z    INFO    controller-runtime.metrics      Metrics server is starting to listen    {"addr": ":8080"}
2022-09-22T09:55:45Z    ERROR   setup   failed to handle audit config   {"namespace": "mondoo-operator", "name": "mondoo-client", "error": "secrets \"mondoo-client-scan-api-token\" not found"}
go.mondoo.com/mondoo-operator/cmd/mondoo-operator/operator.init.0.func1
/home/runner/work/mondoo-operator/mondoo-operator/cmd/mondoo-operator/operator/cmd.go:113
github.com/spf13/cobra.(*Command).execute
/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:872
github.com/spf13/cobra.(*Command).ExecuteC
/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:990
github.com/spf13/cobra.(*Command).Execute
/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:918
main.main
/home/runner/work/mondoo-operator/mondoo-operator/cmd/mondoo-operator/main.go:21
runtime.main
/opt/hostedtoolcache/go/1.19.0/x64/src/runtime/proc.go:250
2022-09-22T09:55:45Z    ERROR   setup   failed to preload scan API URLs {"error": "secrets \"mondoo-client-scan-api-token\" not found"}
github.com/spf13/cobra.(*Command).execute
/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:872
github.com/spf13/cobra.(*Command).ExecuteC
/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:990
github.com/spf13/cobra.(*Command).Execute
/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:918
main.main
/home/runner/work/mondoo-operator/mondoo-operator/cmd/mondoo-operator/main.go:21
runtime.main
/opt/hostedtoolcache/go/1.19.0/x64/src/runtime/proc.go:250
Error: secrets "mondoo-client-scan-api-token" not found
Usage:
mondoo-operator operator [flags]

Flags:
--health-probe-bind-address string   The address the probe endpoint binds to. (default ":8081")
-h, --help                               help for operator
--leader-elect                       Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager.
--metrics-bind-address string        The address the metric endpoint binds to. (default ":8080")

panic: secrets "mondoo-client-scan-api-token" not found

goroutine 1 [running]:
main.main()
        /home/runner/work/mondoo-operator/mondoo-operator/cmd/mondoo-operator/main.go:22 +0xab
```

**To Reproduce**
Steps to reproduce the behavior:
1. Install operator v1.5.0
2. Wait a moment
3. Check operator Pod
4. See error

**Expected behavior**
Operator starts up without problem.

**Screenshots**
-

**Additional context**
-

--- a/cmd/mondoo-operator/operator/cmd.go
+++ b/cmd/mondoo-operator/operator/cmd.go
@@ -1,7 +1,6 @@
 package operator
 
 import (
-	"context"
 	"fmt"
 
 	"github.com/go-logr/logr"
@@ -24,7 +23,6 @@ import (
 	certmanagerv1 "github.com/jetstack/cert-manager/pkg/apis/certmanager/v1"
 	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
 
-	"go.mondoo.com/mondoo-operator/api/v1alpha2"
 	k8sv1alpha2 "go.mondoo.com/mondoo-operator/api/v1alpha2"
 	"go.mondoo.com/mondoo-operator/controllers"
 	"go.mondoo.com/mondoo-operator/controllers/integration"
@@ -35,7 +33,6 @@ import (
 	"go.mondoo.com/mondoo-operator/pkg/utils/logger"
 	"go.mondoo.com/mondoo-operator/pkg/utils/mondoo"
 	"go.mondoo.com/mondoo-operator/pkg/version"
-	"sigs.k8s.io/controller-runtime/pkg/client/config"
 	//+kubebuilder:scaffold:imports
 )
 
@@ -110,10 +107,6 @@ func init() {
 
 		scanApiStore := scan_api_store.NewScanApiStore(ctx)
 		go scanApiStore.Start()
-		if err := preloadScanApiUrls(scanApiStore, scheme, setupLog); err != nil {
-			setupLog.Error(err, "failed to preload scan API URLs")
-			return err
-		}
 		if err = (&controllers.MondooAuditConfigReconciler{
 			Client:                 mgr.GetClient(),
 			MondooClientBuilder:    controllers.MondooClientBuilder,
@@ -189,32 +182,4 @@ func preflightApiChecks(log logr.Logger) (bool, error) {
 	}
 
 	return true, nil
-}
-
-func preloadScanApiUrls(scanApiStore scan_api_store.ScanApiStore, scheme *runtime.Scheme, log logr.Logger) error {
-	cfg, err := config.GetConfig()
-	if err != nil {
-		log.Error(err, "unable to get k8s config")
-		return err
-	}
-
-	kubeClient, err := client.New(cfg, client.Options{Scheme: scheme})
-	if err != nil {
-		log.Error(err, "unable to create k8s client")
-		return err
-	}
-
-	ctx := context.Background()
-	auditConfigs := &v1alpha2.MondooAuditConfigList{}
-	if err := kubeClient.List(ctx, auditConfigs); err != nil {
-		return err
-	}
-
-	for _, auditConfig := range auditConfigs.Items {
-		if err := scan_api_store.HandleAuditConfig(ctx, kubeClient, scanApiStore, auditConfig); err != nil {
-			log.Error(err, "failed to handle audit config", "namespace", auditConfig.Namespace, "name", auditConfig.Name)
-			return err
-		}
-	}
-	return nil
 }

